docs/main/faqs/faqs.md

```diff
@@ -114,44 +508,44 @@ It is recommended to use unpartitioned raw block devices for best results.
 
 [Go to top](#top)
 
 ### How is it configured?
 
-This documentation contains sections which are focused on initial quickstart deployment scenarios, including the correct configuration of underlying hardware and software, and of replicated engine features such as "Storage Nodes" \(MSNs\) and "Disk Pools" \(MSPs\). Information describing tuning for the optimization of performance is also provided.
+This documentation contains sections which are focused on initial quickstart deployment scenarios, including the correct configuration of underlying hardware and software, and of Replicated Storage features such as "Storage Nodes" \(MSNs\) and "Disk Pools" \(MSPs\). Information describing tuning for the optimization of performance is also provided.
 
-* [Quickstart Guide](../quickstart-guide/)
-* [Performance Tips](../user-guides/replicated-engine-user-guide/additional-information/performance-tips.md)
+* [Quickstart Guide](../quickstart-guide/installation.md)
+* [Performance Tips](../user-guides/replicated-storage-user-guide/additional-information/performance-tips.md)
 
 [Go to top](#top)
 
 ### What is the basis for its performance claims? Has it been benchmarked?
 
-Replicated engine has been built to leverage the performance potential of contemporary, high-end, solid state storage devices as a foremost design consideration. For this reason, the I/O path is predicated on NVMe, a transport which is both highly CPU efficient and which demonstrates highly linear resource scaling. The data path runs entirely within user space, also contributing efficiency gains as syscalls are avoided, and is both interrupt and lock free.
+Replicated Storage has been built to leverage the performance potential of contemporary, high-end, solid state storage devices as a foremost design consideration. For this reason, the I/O path is predicated on NVMe, a transport which is both highly CPU efficient and which demonstrates highly linear resource scaling. The data path runs entirely within user space, also contributing efficiency gains as syscalls are avoided, and is both interrupt and lock free.
 
-MayaData has performed its own benchmarking tests in collaboration with Intel, using latest generation Intel P5800X Optane devices "The World's Fastest Data Centre SSD". In those tests it was determined that, on average, across a range of read/write ratios and both with and without synchronous mirroring enabled, the overhead imposed by the replicated engine's I/O path was well under 10% \(in fact, much closer to 6%\).
+MayaData has performed its own benchmarking tests in collaboration with Intel, using latest generation Intel P5800X Optane devices "The World's Fastest Data Centre SSD". In those tests it was determined that, on average, across a range of read/write ratios and both with and without synchronous mirroring enabled, the overhead imposed by the Replicated Storage's I/O path was well under 10% \(in fact, much closer to 6%\).
```

Comment:
Maintainers of Mayastor (Replicated Storage) have performed benchmarking tests in collaboration with Intel...